### ISSUE TITLE
Make it possible to run with no args passed and start recording no questions asked

### ISSUE THREAD

**Issue by kompot** [kompot/nhl-tv-geeky-streams#15]
Could be useful for running on media servers when, say, you have favourite teams selected and would like your team's favourite matches be downloaded in background and be available whenever you're ready to watch them.

**Comment by Redmax87** [comment]
How put my user and password in .yaml file?

**Comment by kompot** [comment]
No problem. Comments/suggestions are welcome.

**Comment by Redmax87** [comment]
Last night I take live NJD@PHI but I did not know how to stop when game end.
What command need?

**Comment by kompot** [comment]
There's no such option (yet?). You can just press ctrl + c and recording will stop.
For archive matches it will stop by itself.

This is not so trivial as it may seem - because one might like to watch live broadcast as long as it lasts and not stop when game finishes.

**Comment by Redmax87** [comment]
And file does not delet, I hope after ctr+c.

**Comment by kompot** [comment]
No, file remains as it is. And actually you can press ctrl+c anytime during broadcast and then run again - recording will continue where it stopped. In case it didn't go from live to archive, in that case it will start from beginning because live and archive are totally different streams. And filename will have `archive` suffix instead of `live`.

**Comment by Redmax87** [comment]
Ok, thanks again, great application, have a good day!
